### DISCUSSION TITLE
The requested module 'https://cdn.skypack.dev/htm/preact/standalone' does not provide an export named 'cloneElement'

### DISCUSSION THREAD

**Discussion by zim32** [preactjs/preact#3804] [Q&A]
This is intentional or bug?

**Comment by rschristian** [comment]
Oversight, likely.

`htm` is another project though. You can find it here: https://github.com/developit/htm

If you wanted to PR a fix, should be super easy to do so. Here's the module that needs adjustment: https://github.com/developit/htm/blob/master/src/integrations/preact/standalone.mjs

**Reply by zim32** [reply]
Maybe just export * from preact?

**Reply by rschristian** [reply]
There's probably a reason why that wasn't done, but I'm not privy to it.
